### ISSUE TITLE
Nuevo estudio: Alusor Stroke

### ISSUE THREAD

**Issue by afk-mario** [indi-es/estudios#126]
## Gracias a: afk - afk@ellugar.co

```json
{
  "name": "Alusor Stroke",
  "country": "México",
  "city": "Guadalajara",
  "state": "Jalisco",
  "website": "http://alusorstroke.com",
  "twitter": "",
  "facebook": "https://www.facebook.com/alusorstroke",
  "instagram": "",
  "tags": [],
  "inactive": false,
  "last_time_active": "2021-11-02T21:22:17.389Z"
}
```

**Comment by Ucamo** [comment]
Part of #133

**Comment by afk-mario** [comment]
closed #133
